kong/runloop/balancer.lua

```diff
@@ -367,10 +367,18 @@ do
         checks.active.unhealthy.interval = 0
       end
 
+      local ssl_cert, ssl_key
+      if kong.configuration.client_ssl then
+        ssl_cert = kong.configuration.client_ssl_cert
```

Comment:
We also support overriding client certificate used for upstream handshake on a per service basis. This should be supported in the health checker as well (possibly involving adding new field to the "Upstream" entity).

https://docs.konghq.com/2.0.x/admin-api/#add-service

Reply:
Fixed!